Week2/moreStrings.js

```diff
@@ -0,0 +1,8 @@
+console.log(`------ 7 ------`);
+
+let myString = `this is a test`;
```

Comment:
Do you think it's better to use `let` or `const` here?

Reply:
Oh, I guess const is better to use since I don't need to assign a new value to it later on. Thanks for the remainder.

Reply:
Again the corrected version is the one with the commit "updating moreStrings.js". Sorry for the confusion..